book.common/src/main/java/com/happystays/book/common/dto/Pnr.java

```diff
@@ -0,0 +1,27 @@
+package com.happystays.book.common.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Pnr {
+
+    private Supplier supplier;
+
+    private Date creationDate;
+
+    private String status;
+
+    private Trip trip;
+
+    private List<HotelInfo> hotelInfoList;
+
+    private List<String> users;
```

Comment:
what is List<String> users ? Is it the user id? id yes please rename as userId or memberId